### PR TITLE
added dcc compatibility, pypi packaged, inheritable

### PR DESCRIPTION
**Pull request by nzanepro**
mega pull request with refactor and lots of features

### CODE REVIEW

**Comment by fredrikaverpil** [comment]
This is great. Will review later tonight and we can merge this shortly.

File: README.md [hunk 1]
```diff
@@ -33,43 +82,44 @@ A boilerplate for creating PyQt4/PySide and PyQt5/PySide2 applications running i
 
 #### Installation
 
-Clone the git repository:
-
+easy way:
 ```bash
-git clone https://github.com/fredrikaverpil/pyvfx-boilerplate.git
+pip install git+https://github.com/nzanepro/pyvfx-boilerplate
```
Comment (fredrikaverpil) [inline README.md]: Change this to say `https://github.com/fredrikaverpil/pyvfx-boilerplate`

File: README.md [hunk 2]
```diff
@@ -33,43 +82,44 @@ A boilerplate for creating PyQt4/PySide and PyQt5/PySide2 applications running i
...
+```
+long way:
+```bash
+git clone https://github.com/nzanepro/pyvfx-boilerplate
```
Comment (fredrikaverpil) [inline README.md]: Change this to say `https://github.com/fredrikaverpil/pyvfx-boilerplate.git`

File: README.md [hunk 3]
```diff
@@ -97,7 +147,7 @@ if QtCompat__binding == 'PySide2':
 
 Something wrong, have a question or wish to file a feature request?
 
-Open up an issue [here](https://github.com/fredrikaverpil/pyvfx-boilerplate/issues)!
+Open up an issue [here](https://github.com/nzanepro/pyvfx.boilerplate/issues)!
```
Comment (fredrikaverpil) [inline README.md]: Change this to say `https://github.com/fredrikaverpil/pyvfx-boilerplate/issues`

File: README.md [hunk 4]
```diff
@@ -1,11 +1,60 @@
...
+pyvfx.boilerplate
 ==================
 
-A boilerplate for creating PyQt4/PySide and PyQt5/PySide2 applications running in Maya, Nuke or completely standalone.
+A boilerplate for creating PyQt4/PySide and PyQt5/PySide2 applications running in Maya, Nuke, Blender, 3DS Max, Houdini, Unreal Engine or completely standalone.
 
 ## Documentation
 
-### Version 2.x
+### Version 3.3.0
```
Comment (fredrikaverpil) [inline README.md]: I think we should break out the version details into a CHANGELOG.md for now, and link to that from the main README.md.

File: README.md [hunk 5]
```diff
@@ -1,11 +1,60 @@
-pyvfx-boilerplate
+pyvfx.boilerplate
```
Comment (fredrikaverpil) [inline README.md]: Why change this to using a period instead of a dash?

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,41 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+name = "pyvfx-boilerplate"
+author = "Fredrik Averpil"
+author_email = "fredrik@averpil.com"
```
[the remark this answers is not in the document]
Comment (fredrikaverpil) [reply setup.py]: Yeah, according to the [official docs](https://docs.python.org/3/distutils/setupscript.html#additional-meta-data) this is not needed, so let's remove author and email.

**Comment by listyque** [comment]
Wow, looks really useful!

**Comment by fredrikaverpil** [comment]
I'm closing this and continuing the work over here: https://github.com/fredrikaverpil/pyvfx-boilerplate/pull/19

**Pull request closed without merging**